presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -440,6 +440,13 @@ protocol::TaskInfo PrestoTask::updateInfoLocked() {
 
   std::unordered_map<std::string, RuntimeMetric> taskRuntimeStats;
 
+  if (taskStats.memoryReclaimCount > 0) {
```

Comment:
It would be nice to report these stats live while query is running. 

There is code later that calls processTaskStats which clears taskRuntimeStats if query is not running.

Also, I noticed that Coordinator UI shows stats based on unit. Hence, you need to specify kNanos unit to ensure nice display.

For example,

```
  taskRuntimeStats.insert(
      {"nativeProcessCpuTime",
       RuntimeMetric(processCpuTime_, RuntimeCounter::Unit::kNanos)});
```

Reply:
You mean to call processTaskStats with tryToSkipIfRunning to false?